### DISCUSSION TITLE
HMS Installation

### DISCUSSION THREAD

**Discussion by BLun78** [DennisOSRM/hms-mqtt-publisher#128] [Q&A]
Hallo,

ich habe mein HMS-800W-2T bisher nicht mit der Cloud verbunden.
Wie muss ich den Konfigurieren damit ich die IP herrausbekomme?
Was ist notwendig um mit deiner Software arbeiten zu könnne? 
Danke für deine bisher Tolle Arbeit.

**Comment by DennisOSRM** [comment]
Du musst den inverter mit dem WLAN verbinden. Das geht mit der S-Miles installer app.
